apps/api/src/app/integrations/e2e/update-integration.e2e.ts

```diff
@@ -55,7 +55,9 @@ describe('Update Integration - /integrations/:integrationId (PUT)', function ()
     // update integration
     await session.testAgent.put(`/v1/integrations/${integrationId}`).send(payload);
 
-    const integration = (await session.testAgent.get(`/v1/integrations`)).body.data[0];
```

Comment:
With the recent changes to the indexing behaviour, it is no longer guaranteed that the first item in the array is the updated one.